nexus/tests/integration_tests/console_api.rs

```diff
@@ -206,6 +209,40 @@ async fn test_absolute_static_dir() {
     cptestctx.teardown().await;
 }
 
+#[nexus_test]
+async fn test_session_me(cptestctx: &ControlPlaneTestContext) {
+    let testctx = &cptestctx.external_client;
+
+    // hitting /session/me without being logged in is a 401
+    RequestBuilder::new(&testctx, Method::GET, "/session/me")
+        .expect_status(Some(StatusCode::UNAUTHORIZED))
+        .execute()
+        .await
+        .expect("failed to 401 on unauthed request");
+
+    // now make same request with auth
+    let priv_user = NexusRequest::object_get(testctx, "/session/me")
+        .authn_as(AuthnMode::PrivilegedUser)
+        .execute()
+        .await
+        .expect("failed to get current user")
+        .parsed_body::<views::User>()
+        .unwrap();
+
+    assert_eq!(priv_user.identity.name.as_str(), "test-privileged");
```

Comment:
There's a global you can use to avoid hardcoding the "test-privileged" string here.

Reply:
Done in 825a8d3861f384b0a7f13933982242fae836d3a4